app/models/concerns/weaknesses/scopes.rb

```diff
@@ -21,6 +21,18 @@ module Weaknesses::Scopes
       ).order(risk: :desc, state: :asc)
     }
     scope :latest, -> { where latest_id: nil }
+
+    def process_owner_parents
+      process_owners.map do |po|
```

Comment:
Y viendo como es ahora, podés poner:
```suggestion
      process_owners.map(&:parent).compact
```

Y listo.